tools/test-locked-requirements

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import difflib
+import filecmp
+import os
+import subprocess
+import sys
+
+from typing import Text
+
+TOOLS_DIR = os.path.dirname(__file__)
+ZULIP_PATH = os.path.dirname(TOOLS_DIR)
+sys.path.append(ZULIP_PATH)
+from scripts.lib.zulip_tools import run
+
+PROD_LOCK_FILE = os.path.join(ZULIP_PATH, 'requirements', 'prod_lock.txt')
+DEV_LOCK_FILE = os.path.join(ZULIP_PATH, 'requirements', 'dev_lock.txt')
+TEST_PROD_LOCK_FILE = '/var/tmp/test_prod_lock.txt'
+TEST_DEV_LOCK_FILE = '/var/tmp/test_dev_lock.txt'
+
+def print_diff(path_file1, path_file2):
+    # type: (Text, Text) -> None
+    with open(path_file1) as file1:
+        with open(path_file2) as file2:
+            diff = difflib.unified_diff(
+                file1.readlines(),
+                file2.readlines(),
+                fromfile=path_file1,
+                tofile=path_file2,
+            )
+    for line in diff:
+        print(line)
+
+def main():
+    # type: () -> None
+    # `pip-compile` tries to avoid unnecessary updation of recursive dependencies
```

Comment:
"unnecessary updation" -> "unnecessarily updating" for correct English.  "updation" isn't a word outside India :) (https://en.wiktionary.org/wiki/updation).